src/CoreBundle/private/library/classes/TCMSListManager/TCMSListManagerFullGroupTable.class.php

```diff
@@ -441,6 +433,37 @@ public function AddFields()
         $this->AddCustomColumns();
     }
 
+    /**
+     * @param array $fieldConfig
+     *
+     * @return null|array|string - returns the types AddColumn allows for the callback parameter
```

Comment:
What are allows?

Reply:
null|array|string like it says. :-) But i added an example for the values. Can't change AddColumn because of BC breaks. So we need to live with this mixed return values. The 2 new methods makes it more readable.